modules/c++/sys/source/OSWin32.cpp

```diff
@@ -100,7 +100,7 @@ sys::Pid_T sys::OSWin32::getProcessId() const
 
 void sys::OSWin32::removeFile(const std::string& pathname) const
 {
-    if (DeleteFile(pathname.c_str()) != true)
+    if (DeleteFile(pathname.c_str()) != TRUE)
```

Comment:
I am assuming that windows.h guarantees there's something called `TRUE` set to non-zero?